client/blocks/plugins-update-manager/schedule-create.tsx

```diff
@@ -34,9 +36,15 @@ export const ScheduleCreate = ( props: Props ) => {
 				<ScheduleForm siteSlug={ siteSlug } />
 			</CardBody>
 			<CardFooter>
-				<Button form="schedule" type="submit" variant="primary">
+				<Button form="schedule" type="submit" variant="primary" disabled={ ! canCreateSchedules }>
 					Create
 				</Button>
+				{ ! canCreateSchedules && (
+					<Text as="p">
+						<Icon className="icon-info" icon={ warning } size={ 16 } />
+						This site cannot auto-update plugins.
```

Comment:
I feel like we might want to emphasize the word "schedule" here. Perhaps something like "This site is unable to schedule auto-updates for plugins." but not a blocker here.

Reply:
Let's also add why the site can't updates so users have an understanding how to fix it

Reply:
I'm generally a proponent of ship and iterate, but for some reason user-facing strings feel different for me. Maybe I've seen them fall through the cracks too many times.

Reply:
I have changed this to `This site is unable to schedule auto-updates for plugins.`

@obenland Could you suggest how users can fix these? From my general googling of the problem the possible causes & solutions are many, so perhaps we should write a separate support doc ourselves.

Reply:
In the endpoint you'll write, let's return a WP error, [similar to what Jetpack does in the plugins endpoint](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php#L298-L341). We can probably even re-use those messages.

Reply:
New endpoint here: D140894-code I decided not to return `WP_Error` as it doesn't seem to be a client or a server error, just an application error state.